### PR TITLE
PORTALS-2985

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Depends on https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/902

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "react-transition-group": "2.6.0",
     "sass": "^1.63.6",
     "spark-md5": "^3.0.2",
-    "synapse-react-client": "3.2.9",
+    "synapse-react-client": "3.2.10",
```
Comment (jay-hodgson) [inline package.json:23]: Contains the "Donate" orientation banner asset

File: src/main/java/org/sagebionetworks/web/client/widget/header/HeaderViewImpl.java [hunk 2]
```diff
@@ -84,6 +89,18 @@ public HeaderViewImpl(
       presenter.onNIHNotificationDismissed();
     });
 
+    donationBanner.configure(
+      "Donate",
+      "Support Open Science and Radical Collaboration with Sage Bionetworks",
+      "Join us as we advance collaborative biomedical research tackling today's most pressing health challenges. Your contribution is crucial to breaking down barriers and accelerating the creation of transformative treatments and technologies. Thank you for being a part of this vital mission and helping us drive innovation forward.",
+      "Donate to Sage",
+      event -> {
+        Window.open("https://sagebionetworks.org/donate", "_blank", "");
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/header/HeaderViewImpl.java:98]: On click, open a new tab to this page on Sagebionetworks.org

File: src/main/java/org/sagebionetworks/web/client/widget/header/HeaderViewImpl.java [hunk 3]
```diff
@@ -210,5 +227,6 @@ public void setPortalAlertVisible(boolean visible, JSONObjectAdapter json) {
   @Override
   public void setNIHAlertVisible(boolean visible) {
     nihNotificationAlert.setVisible(visible);
+    donationBannerContainer.setVisible(!visible);
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/header/HeaderViewImpl.java:230]: This approach does not scale easily to 3+ banners, but we recognize that the Home Page is about to have a redesign (so I don't want to invest in too much at this time)

**Review by hallieswan — APPROVED** [review]

**Pull request merged**